posthog/models/cohort.py

```diff
@@ -61,6 +61,24 @@ class Cohort(models.Model):
 
     objects = CohortManager()
 
+    def get_analytics_metadata(self):
+        action_groups_count: int = 0
```

Comment:
Nit: imo cleaner:

```
action_groups_count = len(1 for group in self.groups if group.has("action_id"))
```

Reply:
Agreed it's cleaner, but then we're doing `2n` instead of `n` loop sequences because we would need one for each action & properties. I did optimize the code a bit based on this suggestion.